pyproject.toml

```diff
@@ -9,7 +9,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.11"
+python = "^3.7"
```

Comment:
The problem here is that we would have errors related to the type hint which is not retro compatible. If you want to work on it fine, else this PR will break everything.

Reply:
That's why i wrote 
> Depends on: https://github.com/aless10/pylings/issues/6

We need to fix that first, don't know how tbh